### ISSUE TITLE
About train

### ISSUE THREAD

**Issue by lml125** [ttxskk/AiOS#5]
How to train your model?thank you

**Comment by WYJSJTU** [comment]
Sorry, due to the license issue, the release of the training dataset *.npz is still in discussion, which is essential for training. But we are expected to release the training code and the preprocessed *.npz files or preprocessing method at the end of May.

**Comment by imabackstabber** [comment]
> Sorry, due to the license issue, the release of the training dataset *.npz is still in discussion, which is essential for training. But we are expected to release the training code and the preprocessed *.npz files or preprocessing method at the end of May.

Hi, any progress about training code or datasets?

**Comment by ttxskk** [comment]
Hi @imabackstabber ,

Thanks for your patience. We recently fixed some errors in the npz files mentioned in issue #10, resulting in improved performance on AGORA compared to our paper. However, we still need to conduct further validation. I will release all the files once the validation is complete. But if you need these npz files now, please feel free to let me know, and I can share a copy here.

**Comment by imabackstabber** [comment]
> Hi @imabackstabber ,
> 
> Thanks for your patience. We recently fixed some errors in the npz files mentioned in issue #10, resulting in improved performance on AGORA compared to our paper. However, we still need to conduct further validation. I will release all the files once the validation is complete. But if you need these npz files now, please feel free to let me know, and I can share a copy here.

Sure, I'll be happy if I could get one copy! Could you share it with a google drive link or something?

**Comment by hengyuan-zhang-0** [comment]
Thank you for your contribution. I would be incredibly grateful if you could kindly provide me with a copy as well.

**Comment by ttxskk** [comment]
Hi @imabackstabber @hengyuan-zhang-0, I will update it in this PR #15.

**Comment by ttxskk** [comment]
Hi @imabackstabber @hengyuan-zhang-0, #15 has been merged into the main branch.  If you encounter any issues, please feel free to reach out. If I don't respond in time, you can also email me at qingping.sun@my.cityu.edu.hk.

**Comment by Maskros-new** [comment]
Thanks for ur great work!!

Error about training:
I use the bash 'sh scripts/train.sh' for training, but an error occurred.

[08/31 00:21:28.920]: Creating dataset...
[AGORA_MM] Cache not found, generating cache...
Traceback (most recent call last):
  File "main.py", line 388, in <module>
    main(args)
  File "main.py", line 188, in main
    eval(trainset_i)(transforms.ToTensor(), 'train'))
  File "/data1/data1/jhh/workspace/AiOS_new/AiOS-main/datasets/AGORA_MM.py", line 75, in __init__
    self.datalist = self.load_data(
  File "/data1/data1/jhh/workspace/AiOS_new/AiOS-main/datasets/AGORA_MM.py", line 83, in load_data
    content = np.load(self.annot_path, allow_pickle=True)
  File "/mnt/data1/jhh/miniconda3/envs/closeint/lib/python3.8/site-packages/numpy/lib/npyio.py", line 407, in load
    fid = stack.enter_context(open(os_fspath(file), "rb"))
NotADirectoryError: [Errno 20] Not a directory: 'data/preprocessed_npz/cache/agora_train_3840_w_occ_multi_2010.npz'

I have downloaded the preprocessed_npz and put in the correct place. Can anyone who has trained a model answer this question for me? Thanks!!

**Comment by ttxskk** [comment]
Hi @Maskros-new, Could you please check if `agora_train_3840_w_occ_multi_2010.npz` has been placed under `data/preprocessed_npz/cache`? From your log, I guess the npz file is not placed under the correct path.

**Comment by ttxskk** [comment]
Closed. If you have any other questions, please reopen it.
